app/views/crystals/index.html.erb

```diff
@@ -1,7 +1,8 @@
+<a href="/crystal_sets">Back to Crystal Sets</a>
```

Comment:
If you look in the view layouts folder and open the application.html.erb file you'll have access to the layout of every single page you create. If a link needs to be on every page you can add it above <%= yield %> to have it show up on any page you create in the future. In addition you can edit a single area to change al the links at once.

Reply:
!!!!!!! THANK YOU!!!!!!!!!!